### DISCUSSION TITLE
Using unmerged arrow-rs PRs in Datafusion

### DISCUSSION THREAD

**Discussion by metesynnada** [apache/datafusion#5439] [Q&A] [answered]
I want to develop new features depending on the changes I made in `arrow-rs`. Normally I would put `arrow = { git = "https://github.com/synnada-ai/arrow-rs.git", rev = "asdasd" }` in `Cargo.toml`, however, I have to change every arrow import in the cargo file. 

Is there a shortcut for this?

**Comment by tustvold** [comment, marked as answer]
You can use a patch, e.g. https://github.com/apache/arrow-datafusion/pull/5441/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R55
